### PR TITLE
 darksidemanas.md

### PR DESCRIPTION
**Pull request by darksidemanas**
about a revolution in the health industry

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by darksidemanas** [comment]
hope to work on this project

File: darksidemanas.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Project Name
+MEDICAL SOS EMERGENCY MONITORING AND RECORDING SYSTEM 
+## Summary
+
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
+I WILL BE DEVEOPING ANDRIOD APP TO DO SO.
+
+## Plan
+
+1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTON DESINGED PCB
```
Comment (bellesea) [inline darksidemanas.md:15]:
```suggestion
1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTOM DESIGNED PCB 
```

File: darksidemanas.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+MEDICAL SOS EMERGENCY MONITORING AND RECORDING SYSTEM 
+## Summary
+
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
+I WILL BE DEVEOPING ANDRIOD APP TO DO SO.
+
+## Plan
+
+1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTON DESINGED PCB 
+2.TO MAKE A ANDIOD APP WITH MULTIPLE LANGUAGE SUPPORT AN ALGORITHM TO LIST OUT PATIENTS WHO NEED MEDICAL ASSISTANCE ARGENTLY
```
Comment (bellesea) [inline darksidemanas.md:16]:
```suggestion
2.TO MAKE AN ANDRIOD APP WITH MULTIPLE LANGUAGE SUPPORT AN ALGORITHM TO LIST OUT PATIENTS WHO NEED MEDICAL ASSISTANCE URGENTLY 
```

File: darksidemanas.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Summary
+
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
+I WILL BE DEVEOPING ANDRIOD APP TO DO SO.
+
+## Plan
+
+1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTON DESINGED PCB 
+2.TO MAKE A ANDIOD APP WITH MULTIPLE LANGUAGE SUPPORT AN ALGORITHM TO LIST OUT PATIENTS WHO NEED MEDICAL ASSISTANCE ARGENTLY 
+3.PCB TOH CONNECT DIFFRENT MEDICAL PROBES WITH PI
```
Comment (bellesea) [inline darksidemanas.md:17]:
```suggestion
3.PCB TOHCONNECT DIFFERENT MEDICAL PROBES WITH PI 
```

File: darksidemanas.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
+I WILL BE DEVEOPING ANDRIOD APP TO DO SO.
+
+## Plan
+
+1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTON DESINGED PCB 
+2.TO MAKE A ANDIOD APP WITH MULTIPLE LANGUAGE SUPPORT AN ALGORITHM TO LIST OUT PATIENTS WHO NEED MEDICAL ASSISTANCE ARGENTLY 
+3.PCB TOH CONNECT DIFFRENT MEDICAL PROBES WITH PI 
+4.TESTING WITH DIFFRENT MEDICAL SENSORS
```
Comment (bellesea) [inline darksidemanas.md:18]:
```suggestion
4.TESTING WITH DIFFERENT MEDICAL SENSORS 
```

File: darksidemanas.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
+I WILL BE DEVEOPING ANDRIOD APP TO DO SO.
+
+## Plan
+
+1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTON DESINGED PCB 
+2.TO MAKE A ANDIOD APP WITH MULTIPLE LANGUAGE SUPPORT AN ALGORITHM TO LIST OUT PATIENTS WHO NEED MEDICAL ASSISTANCE ARGENTLY 
+3.PCB TOH CONNECT DIFFRENT MEDICAL PROBES WITH PI 
+4.TESTING WITH DIFFRENT MEDICAL SENSORS 
+5.DESINGING A METAL ENCLOSER TO PROTECT PI AND OTHER PCB .
```
Comment (bellesea) [inline darksidemanas.md:19]:
```suggestion
5.DESIGNING A METAL ENCLOSURE TO PROTECT PI AND OTHER PCB .
```

File: darksidemanas.md (hunk 3 above)
Comment (bellesea) [reply darksidemanas.md:17]: Could you please elaborate on what you mean by medical probes? Are there any specific ones you're thinking of and do you already have access to them?

File: darksidemanas.md (hunk 5 above)
Comment (bellesea) [reply darksidemanas.md:19]: how do you plan on building this metal enclosure?

File: darksidemanas.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| SOLDRING KIT    | https://robu.in/product/plusivo-soldering-kit-with-multimeter-eu-plug/ | $21.27  |
+| raspberyy pi 4B+ 8GB| https://www.amazon.in/Raspberry-Pi-8GB-Desktop-Computer/dp/B08B9XS3B6?th=1  | $169.18 |
```
Comment (bellesea) [inline darksidemanas.md:28]: do you need an SD card and power adapter? also, how do you plan on connecting to and programming your raspberry pi?

File: darksidemanas.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+---
+name: "darksidemanas"
+project: "medical sos emergeny recordig and monitoring system"
+---
+
+# Project Name
+MEDICAL SOS EMERGENCY MONITORING AND RECORDING SYSTEM 
+## Summary
+
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
```
Comment (bellesea) [inline darksidemanas.md:10]: in order for your device to monitor multiple people at a time, would you need to have multiple devices?

File: darksidemanas.md (hunk 1 above)
Comment (bellesea) [reply darksidemanas.md:15]: do you plan on following any guide of sorts for designing your pcb such that your pi can read analog signals?

**Review by bellesea — CHANGES_REQUESTED** [review]
@darksidemanas thank you so much for sharing your project idea! i left some questions in the file :)) thank you sm!

File: darksidemanas.md (hunk 1 above)
Comment (darksidemanas) [reply darksidemanas.md:15]: yup we can use the mcb3008 module to help pi to read analogue data

File: darksidemanas.md (hunk 3 above)
Comment (darksidemanas) [reply darksidemanas.md:17]: medical probes suggest equipment like ECGs and other present in ICU or NICUS

File: darksidemanas.md (hunk 6 above)
Comment (darksidemanas) [reply darksidemanas.md:28]: I have both with me I m thinking of using SSH with LAN cable and with putty

File: darksidemanas.md (hunk 7 above)
Comment (darksidemanas) [reply darksidemanas.md:10]: to monitor every person we need a different device

File: darksidemanas.md (hunk 6 above)
Comment (bellesea) [reply darksidemanas.md:28]: sounds good!

File: darksidemanas.md (hunk 7 above)
Comment (bellesea) [reply darksidemanas.md:10]: ahhh ok :)

File: darksidemanas.md (hunk 5 above)
Comment (bellesea) [reply darksidemanas.md:19]: cool! and do you already have access to an appropriate 3d printer or other resources you need to design the case?

File: darksidemanas.md (hunk 3 above)
Comment (bellesea) [reply darksidemanas.md:17]: oh cool! could you please elaborate a little more on this within your plan (and add it to the file)? like how would you connect the equipment to your machine, what are you going to build in your circuit that will allow it to read the data from these machines? etc.

File: darksidemanas.md (hunk 1 above)
Comment (bellesea) [reply darksidemanas.md:15]: do you already have access to that?

Comment (darksidemanas) [reply darksidemanas.md:15]: yup if you want I can share bills and pictures

File: darksidemanas.md (hunk 3 above)
Comment (darksidemanas) [reply darksidemanas.md:17]: like these machine uses analogue signals to transmit data to monitors and other equipment we will be using the mcb3008 and spi interface present in raspberry pi we will be reading analogue signals .

File: darksidemanas.md (hunk 5 above)
Comment (darksidemanas) [reply darksidemanas.md:19]: yup in my community I have access to 3d printer ,well I m not sure about it is capable of printing metals or not but I am sure about ABS .

Comment (bellesea) [reply darksidemanas.md:19]: sounds good, thanks!

File: darksidemanas.md (hunk 3 above)
Comment (bellesea) [reply darksidemanas.md:17]: ok cool! can you please add this in your file? also, is the budget inclusive of taxes and shipping?

Comment (darksidemanas) [reply darksidemanas.md:17]: ok sure

File: darksidemanas.md (hunk 1 above)
Comment (bellesea) [reply darksidemanas.md]: no need hehe, thanks!

**Comment by darksidemanas** [comment]
hey please share your slack id if you don't mind for quick conversations

**Comment by bellesea** [comment]
@darksidemanas i'm so sorry i completely missed that you already committed the changes!

**Comment by darksidemanas** [comment]
> @darksidemanas i'm so sorry i completely missed that you already committed the changes!

no worries ,I know you are already in alot of load let me know the next step

**Review by bellesea — APPROVED** [review]
@darksidemanas this is awesome! can't wait to see you work on this :))

an awesome member of @hackclub/bank-ops will be reaching out soon with next steps (please note we're off on weekends & this coming monday).

thank you!

**Review by MelSmith104 — COMMENTED** [review]
hey @darksidemanas! i'd love to learn more about this project. it has such a practical use. im dm'ing you on slack right now to chat about next steps :)

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**